### PR TITLE
Add dart compilation rule for service_streaming_proto

### PR DESCRIPTION
**Pull request by rbehjati**

### CODE REVIEW

File: oak_remote_attestation/proto/v1/BUILD [hunk 1]
```diff
@@ -19,6 +19,7 @@ load("@io_grpc_grpc_java//:java_grpc_library.bzl", "java_grpc_library")
 load("@rules_cc//cc:defs.bzl", "cc_proto_library")
 load("@rules_java//java:defs.bzl", "java_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//dart:build_defs.bzl", "dart_proto_library")
```
Comment (ipetr0v) [inline oak_remote_attestation/proto/v1/BUILD:22]: Where is it defined? In the `third_party`?

**Review by ipetr0v — APPROVED** [review]

File: oak_remote_attestation/proto/v1/BUILD (hunk 1 above)
Comment (rbehjati) [reply oak_remote_attestation/proto/v1/BUILD]: Not sure. I am still experimenting to figure out how to load `dart_proto_library`.

Comment (rbehjati) [reply oak_remote_attestation/proto/v1/BUILD]: I cannot find anything dart-related in https://github.com/stackb/rules_proto. Any ideas how to fix this @tiziano88?

**Comment by rbehjati** [comment]
Dart support for proto via bazel is lacking currently.

**Pull request closed without merging**